<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hotel System</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="exitBtn">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>290</y>
      <width>88</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Выход</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="loginFiel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Логин</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Пароль</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="clientRB">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Клиент</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="workerRB">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>90</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Работник</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loginBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>101</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Войти</string>
    </property>
   </widget>
   <widget class="QPushButton" name="regBtn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>101</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Регистрация</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwdField">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>459</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
